src/inference.py

```diff
@@ -25,9 +27,11 @@ def inference(config: Struct):
     if config.model_type.lower() == "longnet":
         model = LongNetModel(config)
```

Comment:
Why doesn't this get changed to `LongNetLightning` as well?

Reply:
This issus is fixed.